lib/backup_restore/backuper.rb

```diff
@@ -321,8 +321,9 @@ def notify_user
       log "Notifying '#{@user.username}' of the end of the backup..."
       status = @success ? :backup_succeeded : :backup_failed
 
+      upload = Logger.save_log_to_upload(user: @user, logs: @logs)
       post = SystemMessage.create_from_system_user(
-        @user, status, logs: Discourse::Utils.pretty_logs(@logs)
+        @user, status, logs: UploadMarkdown.new(upload).attachment_markdown
```

Comment:
Why would happen here if `upload` is not `persisted?`

Reply:
The PM will continue the error message why the upload did not persist.